### PR TITLE
Blog post announcing package score section

### PR DESCRIPTION
**Pull request by cyndichin**
Blog post announcing the work done under the Swift Mentorship Program:
* Creating new package score section in maintainer's page
* Adding two additional score metrics

### CODE REVIEW

**Comment by daveverwer** [comment]
I think we got a slightly older version than the one in the Google doc, so I updated to that version and added the images. This is looking great. Ready for merging next week 👍

File: Content/posts/revealing-and-explaining-package-scores.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The most important thing about this package score is that we want it to
+
+While the algorithm for calculating package scores [has always been publicly available](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/Sources/App/Core/Score.swift), it’s unreasonable to expect people to browse the source code to see how we score packages. Today’s release makes the scoring process much more transparent.
+
+### Score traits
+
+The package score is currently evaluated based on ten score traits. Previously, we relied on eight attributes to determine a package’s score.
+
+- Whether the package is archived or not (max +20 points)
```
Comment (cyndichin) [inline Content/posts/revealing-and-explaining-package-scores.md:36]: Also, do we want to remove `+` if we're using `Up to`
```suggestion
- Whether the package is archived or not (Up to 20 points)
```

File: Content/posts/revealing-and-explaining-package-scores.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+### Finding your package’s score
+
+As a package author, if you’d like to see the breakdown of your package score, find the “Do you maintain this package?” section at the bottom of the right-hand sidebar on your package page, and you’ll find the score breakdown at the end.
+
+<picture class=“shadow“>
+  <source srcset=“/images/find-package-maintainers-page~dark.png“ media=“(prefers-color-scheme: dark)“>
+  <img src=“/images/find-package-maintainers-page~light.png“ alt=“The location of the Learn More link that takes package authors to the package maintainer’s page for their package.“>
+</picture>
+
+If you have any feedback regarding the package score or suggestions on how we can improve it, we would love to hear your thoughts in our [discussion forum](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/discussions/2591). We appreciate any contributions as we continue to enhance the package score system.
```
Comment (cyndichin) [inline Content/posts/revealing-and-explaining-package-scores.md:63]: I also wanted to add this note to the bottom, but let me know if you feel this is better to include in the Swift Forums blog post instead.

> As a final note to this post, I can't express how thankful I am to have had Dave as my mentor in the Swift Mentorship program. Contributing to this project under his guidance has been an incredible learning experience and I'm grateful for the knowledge, support, and inspiration he has provided me. Thank you Dave!

File: Content/posts/revealing-and-explaining-package-scores.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+### What is a package score?
+
+In combination with the relevancy of a search query, we use a package score to partially influence the ordering of search results on the Swift Package Index. The new information gives package authors and maintainers insight into how we calculate package scores, and the metrics that calculation relies on, such as how actively maintained a package is, whether it has documentation, tests, and various other factors.
+
+<picture class="shadow">
+  <source srcset="/images/package-maintainers-score-section~dark.png" media="(prefers-color-scheme: dark)">
+  <img src="/images/package-maintainers-score-section~light.png" alt="The package score section on the package maintainer's page showing a breakdown of package score for a package.">
+</picture>
+
+The most important thing about this package score is that we want it to
```
Comment (cyndichin) [inline Content/posts/revealing-and-explaining-package-scores.md:28]: do we want to continue this sentence, maybe?
`inform package maintainer's on their package score and provide visibility into how it is calculated.`

**Review by cyndichin — COMMENTED** [review]
oh thank you for the update, sounds way better! I added some comments, but we can also merge together during our next meeting :)

File: Content/posts/revealing-and-explaining-package-scores.md (hunk 2 above)
Comment (daveverwer) [reply Content/posts/revealing-and-explaining-package-scores.md:63]: Oops, did I accidentally remove that? 🫨 Sorry. Thank you for the kind words and please feel free to add it back.

File: Content/posts/revealing-and-explaining-package-scores.md (hunk 3 above)
Comment (daveverwer) [reply Content/posts/revealing-and-explaining-package-scores.md:28]: Sounds great 👍

File: Content/posts/revealing-and-explaining-package-scores.md (hunk 2 above)
Comment (cyndichin) [reply Content/posts/revealing-and-explaining-package-scores.md:63]: no problem! updated :)

**Review by daveverwer — APPROVED** [review]

**Pull request merged**